src/extension/options-page/DashboardComponents/AbstractTab.tsx

```diff
@@ -1,50 +1,33 @@
 import React from 'react'
 import { makeStyles } from '@material-ui/styles'
-import { Theme, createStyles, AppBar, Tabs, Tab, Typography, Box, TypographyProps, BoxProps } from '@material-ui/core'
-import classNames from 'classnames'
+import { Theme, createStyles, Tabs, Tab, Typography, Box, TypographyProps, BoxProps, Paper } from '@material-ui/core'
 
 const useStyles = makeStyles((theme: Theme) =>
     createStyles({
-        panelBorder: {
-            border: `1px solid ${theme.palette.type === 'dark' ? 'rgba(1, 1, 1, 0.3)' : 'rgba(33, 33, 33, 0.08)'}`,
-        },
-        panelBorderMarginTop: {
-            marginTop: theme.spacing(2),
-        },
-        panelBorderMarginBottom: {
-            marginBottom: theme.spacing(2),
-        },
-        appbarBackground: {
-            background: theme.palette.type === 'dark' ? 'rgba(1, 1, 1, 0.2)' : 'rgba(33, 33, 33, 0.08)',
-        },
-        tabSelected: {
-            background: theme.palette.type === 'dark' ? 'rgba(18, 12, 40, 0.2)' : 'rgba(237, 243, 254, 0.8)',
-        },
+        tabs: {},
     }),
 )
 
-const TabPanel = (props: TypographyProps & Pick<BoxProps, 'p' | 'height'> & { value: number; index: number }) => {
-    const { children, value, index, p, height, ...other } = props
+interface TabPanelProps extends BoxProps {
+    label: string
+}
 
-    return (
-        <Typography role="tabpanel" hidden={value !== index} {...{ component: 'div' }} {...other}>
-            <Box height={height} p={p ?? 3}>
-                {children}
-            </Box>
-        </Typography>
-    )
+const TabPanel = (props: TabPanelProps & { value: number; index: number }) => {
+    const { value, index, display, label, ...other } = props
```

Comment:
label seems like unused

Reply:
things may still be adjusted... they are not very stable... don't be in a hurry